rubies/ruby/CVE-2015-1855.yml

```diff
@@ -11,7 +11,29 @@ description: |
   follows more strict behavior, as recommended by these RFCs. In particular,
   matching of more than one wildcard per subject/SAN is no-longer allowed. As well,
   comparison of these values is now case-insensitive.
+
+  verify_certificate_identity in the OpenSSL extension in Ruby before
+  2.0.0 patchlevel 645, 2.1.x before 2.1.6, and 2.2.x before 2.2.2
+  does not properly validate hostnames, which allows remote attackers
+  to spoof servers via vectors related to (1) multiple wildcards,
+  (1) wildcards in IDNA names, (3) case sensitivity, and
+  (4) non-ASCII characters.
+cvss_v2: 4.3
+cvss_v3: 5.9
 patched_versions:
-- ~> 2.0.0.645
-- ~> 2.1.6
-- '>= 2.2.2'
+ - '~> 2.0.0-p645'
+ - '~> 2.1.6'
+ - '>= 2.2.2'
```

Comment:
I thought we were standardizing on double-quotes for version ranges? Why single-quotes for these?

Reply:
sync script does double-quotes and I see single-quotes and no quotes in the repo, so I pick singles.